packages/apidom-converter/src/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/security-scheme-type.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  ComponentsElement,
+  OpenApi3_1Element,
+  SecurityRequirementElement,
+  SecuritySchemeElement,
+  isSecuritySchemeElement,
+  isComponentsElement,
+  isSecurityRequirementElement,
+} from '@swagger-api/apidom-ns-openapi-3-1';
+import { AnnotationElement, toValue, isObjectElement } from '@swagger-api/apidom-core';
+
+type SecuritySchemeTypePluginOptions = {
+  annotations: AnnotationElement[];
+};
+
+const securitySchemeTypeRefractorPlugin =
+  ({ annotations }: SecuritySchemeTypePluginOptions) =>
+  () => {
+    const removedSecuritySchemes: SecuritySchemeElement[] = [];
+    const annotation = new AnnotationElement(
+      '"mutualTLS" type is not supported in OpenAPI 3.0.3. It will be removed from the converted document.',
+      { classes: ['error'] },
+      { code: 'mutualTLS' },
+    );
+
+    return {
+      visitor: {
+        OpenApi3_1Element(element: OpenApi3_1Element) {
+          if (!isComponentsElement(element.components)) return undefined;
+          if (!isObjectElement(element.components.securitySchemes)) return undefined;
+
+          element.components.securitySchemes.forEach((value) => {
+            if (isSecuritySchemeElement(value) && toValue(value.type) === 'mutualTLS') {
+              removedSecuritySchemes.push(value);
+            }
+          });
+
+          return undefined;
+        },
+        ComponentsElement(element: ComponentsElement) {
+          if (!isObjectElement(element.securitySchemes)) return undefined;
+
+          element.securitySchemes.forEach((value, key) => {
+            if (isSecuritySchemeElement(value) && toValue(value.type) === 'mutualTLS') {
+              if (!removedSecuritySchemes.includes(value)) removedSecuritySchemes.push(value);
+              (element.securitySchemes as SecuritySchemeElement).remove(toValue(key));
+              annotations.push(annotation);
+            }
+          });
+
+          return undefined;
+        },
+        SecurityRequirementElement(element: SecurityRequirementElement) {
+          if (!removedSecuritySchemes.length || !isSecurityRequirementElement(element))
```

Comment:
```suggestion
          if (!removedSecuritySchemes.length)
```

It it guaranteed that when the `SecurityRequirementElement` hook executes it's only paramter will be of `SecurityRequirementElement` instance. No need for checks here.